### ISSUE TITLE
jsep dependency error while using expression-eval

### ISSUE THREAD

**Issue by jeromecambon** [donmccurdy/expression-eval#41]
From TypeScript ('esnext' module), when trying to use expression-eval:

```
import {parse} from "expression-eval";
  
evalVariable(varName: string) {
    let ast = parse(varName);
    console.log("ast=" + ast);
}
```

I get the following error in the console:
`Uncaught SyntaxError: The requested module '../../jsep/build/jsep.js' does not provide an export named 'default'`

**Comment by donmccurdy** [comment]
Does `import jsep from 'jsep';` work in your project? This may be specific to your build settings, or just an issue in the jsep module...

**Comment by jeromecambon** [comment]
When I use:
```
import jsep from 'jsep';
evalVariable(varName: string) {
    let parseExp = jsep("1 + 1");
    console.log("parseExp=" + parseExp);
}
```
I get:
`Uncaught SyntaxError: The requested module '../../node_modules/jsep/build/jsep.js' does not provide an export named 'default'`

Note that I set typescrit parameters 'esModuleInterop' and 'allowSyntheticDefaultImports' to true.

**Comment by dheerajkhatri** [comment]
@jeromecambon I am also facing the same issue, I modified import statement as:
`import * as jsep from 'jsep';` and it worked fine. 

It got modified here:
https://github.com/donmccurdy/expression-eval/commit/dd14637c10d2a3dace086da61b2c982fb31db1be#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8

**Comment by donmccurdy** [comment]
This change was made in https://github.com/donmccurdy/expression-eval/pull/37, and fixed an error in @rbayliss's project with the previous syntax. Moreover, it matches what we actually do in the source file, `index.js`. 

Since I'm not familiar with your various projects and build configurations, I'm not sure I know how to resolve this for everyone. I'm a bit tempted to remove the TypeScript definitions from this project, since it's difficult to maintain them when the source is not TypeScript, and TypeScript itself provides no robust tooling to check whether they're correct. 😕

**Comment by donmccurdy** [comment]
I believe this is fixed by 4.0.0. I've converted the library to typescript, so the type definitions are no longer written by hand. Note that tsconfig does use `"esModuleInterop": true`.

**Comment by tirithen** [comment]
I get the same error in a plain javascript project when in a test (using web-test-runner) importing with:

```javascript
import { parse, eval } from 'expression-eval';
```
```bash
SyntaxError: The requested module './../../jsep/build/jsep.js' does not provide an export named 'default'
```

Probably this package does not comply fully to ECMAScript modules import/export syntax, but maybe rather the older node/cjs types.

I'm trying an older version of the package to start with as mentioned on some of the other issues around the import errors.
